fractal_tasks_core/cellvoyager/metadata.py

```diff
@@ -174,8 +185,14 @@ def read_mrf_file(mrf_path: str):
     mrf_frame = pd.DataFrame(columns=mrf_columns)
 
     mrf_xml = ElementTree.parse(mrf_path).getroot()
+    # fixme: get bts:ColumnCount="48" bts:RowCount="32"
     # Read mrf file
     ns = {"bts": "http://www.yokogawa.co.jp/BTS/BTSSchema/1.0"}
+    # Fetch RowCount and ColumnCount
+    row_count = int(mrf_xml.attrib[f'{{{ns["bts"]}}}RowCount'])
+    column_count = int(mrf_xml.attrib[f'{{{ns["bts"]}}}ColumnCount'])
+    plate_layout = row_count * column_count
```

Comment:
Is `plate_layout` a common keyword for the number of wells? If not, what about `well_count`?

(I'm just raising the question, but it's clearly a minor issue)

Reply:
Hmm, good question. Thinking about it, what I really mean there is `plate_type`. There is a 1536 well `plate_type` that should be treated differently from others. We usually deal with 96 well & 384 well plates, but some other configurations appear to follow the same well naming conventions, so I wouldn't want to be too strict on our default behavior.
But renaming this to `plate_type` would make sense

Reply:
Done